phpBB/styles/prosilver/theme/common.css

```diff
@@ -1251,6 +1251,14 @@ ul.linklist:after,
 	margin-left: 58px;
 }
 
+.notification-alert,
+.compact .notification-alert {
+	border-radius: 50px;
+	font-size: 0.9em;
+	opacity: 0.8;
+	padding: 4px 7px;
```

Comment:
its because you need to reset the line height, just use BS badge code. https://github.com/twbs/bootstrap/blob/v4-dev/scss/_badge.scss To include the pos:rel with a top of -3 or something like that.